sormas-ui/src/main/java/de/symeda/sormas/ui/campaign/campaigns/CampaignEditForm.java

```diff
@@ -167,6 +171,11 @@ protected void addFields() {
 
 		final Label spacer = new Label();
 		getContent().addComponent(spacer, SPACE_LOC);
+
+		addField(ContactDto.DELETE_REASON);
+		addField(ContactDto.OTHER_DELETE_REASON, TextArea.class).setRows(3);
+		setVisible(false, ContactDto.DELETE_REASON);
```

Comment:
You can combine these into one single call (because `setVisible` has a varargs parameter), also for the other forms. And please make sure to use the correct classes (i.e. in this case CampaignDto instead of ContactDto)

Reply:
changed